Tests/BotTests/GitHub/GitHubAPITests.swift

```diff
@@ -0,0 +1,287 @@
+import XCTest
+import Nimble
+@testable import Bot
+
+class GitHubAPITests: XCTestCase {
+
+    var directory: URL {
+        return URL(string: #file)!
```

Comment:
Curious why you don't use `init(fileURLWithPath: String)` ? With just a string you don't get the `file:///` prefix.

Reply:
Ok, I see you are passing this to `FileManager.default.contents(atPath: url.absoluteString)`, which doesn't need the `file:///` prefix 👍

Reply:
I had that first but was failing while reading, probably I need to use a different API. I can re-test if necessary.